crates/tiny/config.yml

```diff
@@ -21,9 +21,18 @@ servers:
       # pass: 'hunter2'
 
       # SASL authentication
+      # We currently support SASL PLAIN or SASL EXTERNAL
+      # Providing a username and password you will configure SASL PLAIN
+      # Providing a path to a PEM file will configure SASL EXTERNAL
+      # For SASL EXTERNAL certificate and fingerprint generation, see server documentation.
+      # You will need to register the cert's fingerprint with NickServ
+      # ex. https://www.oftc.net/NickServ/CertFP/
```

Comment:
Remove this paragraph please. The config is getting quite large. We have an example below on how to use SASL EXTERNAL. We should add this info to the README if necessary.

Reply:
:+1: moving this to wiki makes sense to me.

Reply:
- [ ] Add to wiki once merged